weather_mv/loader_pipeline/sinks_test.py

```diff
@@ -67,6 +98,15 @@ def test_opens_tif_files(self):
             self.assertIsNotNone(ds)
             self.assertDictContainsSubset({'is_normalized': False}, ds.attrs)
 
+    def test_open_dataset__fits_memory_bounds(self):
```

Comment:
Nice! 

Can you extract this into a decorator or context manager? I think it would be a good test util to have.

Reply:
Added context manager in `test_open_dataset__fits_memory_bounds`.